src/Actions/AttemptToAuthenticate.php

```diff
@@ -103,11 +103,12 @@ protected function throwFailedAuthenticationException($request)
      * Fire the failed authentication attempt event with the given arguments.
      *
      * @param  \Illuminate\Http\Request  $request
+     * @param  \Illuminate\Contracts\Auth\Authenticatable|null  $user
      * @return void
      */
-    protected function fireFailedEvent($request)
+    protected function fireFailedEvent($request, $user = null)
```

Comment:
I'm not sure why you made this change? On both of the places where you call and pass the `$user` you know that they're not set (see the ternary check in the if statement before calling the method). The user will always be `null`.

Reply:
ok, removed!